config/config.default.yaml

```
@@ -693,6 +693,7 @@ sector:
   electricity_distribution_grid: true
   electricity_grid_connection: true
   transmission_efficiency:
+    enable: ['DC', 'H2 pipeline', 'gas pipeline', 'electricity distribution grid']
```

Let's do more YAML-style:
```suggestion
    enable:
    - DC
    - H2 pipeline
    - gas pipeline
    - electricity distribution grid
```